browser-sign-in/OktaBrowserSignIn/TokensViewController.swift

```diff
@@ -109,18 +144,4 @@ private extension TokensViewController {
     func showError(_ message: String) {
         self.showAlert(title: "Error", message: message)
     }
-    
-    func startProgress() {
```

Comment:
Do we no longer show progress / activity during these operations?

Reply:
No we do not, actually, the show progress / activity  was not called in any function previously.